spec/library_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'date'
+require 'pry-byebug'
+require './lib/library.rb'
+
+
+describe Library do
+
+  it 'checks that there is a collection of books' do
+    expect(:book).not_to be_empty
```

Comment:
don't you mean to `:books` here? 

Wonder why this passed? 🤔  - You probably have another variable or method `book` on your `Library` class that evaluates to a non_empty object